packages/flutter/lib/src/widgets/container.dart

```diff
@@ -442,10 +463,34 @@ class Container extends StatelessWidget {
     super.debugFillProperties(properties);
     properties.add(DiagnosticsProperty<AlignmentGeometry>('alignment', alignment, showName: false, defaultValue: null));
     properties.add(DiagnosticsProperty<EdgeInsetsGeometry>('padding', padding, defaultValue: null));
+    properties.add(DiagnosticsProperty<Clip>('clipBehavior', clipBehavior, defaultValue: null));
     properties.add(DiagnosticsProperty<Decoration>('bg', decoration, defaultValue: null));
     properties.add(DiagnosticsProperty<Decoration>('fg', foregroundDecoration, defaultValue: null));
     properties.add(DiagnosticsProperty<BoxConstraints>('constraints', constraints, defaultValue: null));
     properties.add(DiagnosticsProperty<EdgeInsetsGeometry>('margin', margin, defaultValue: null));
     properties.add(ObjectFlagProperty<Matrix4>.has('transform', transform));
   }
 }
+
+// this is used in Container for ClipPath and implementation is closely related
+// to the [Container.clipBehaviour].
+//
+// if there is any plan to make this public, it's better to change the
+// implementation to fit more general use cases.
```

Comment:
Nit: These should use proper grammar/punctuation/capitalization.

 The second paragraph seems unnecessary.